ting-bill-split.go

```diff
@@ -175,6 +197,20 @@ func parseMegabytes(megReader io.Reader) (map[string]int, error) {
 func main() {
 	fmt.Printf("Ting Bill Splitter\n\n")
 
+	f, err := os.Open("bills.toml")
```

Comment:
Instead of specifically looking for `"bills.toml"` let's move all of this code down below the flag parsing and add a flag `-bill` or `-bills` that points to the bill file